builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Moralis
+description: Learn how to build on the Moonbeam network with Moralis' API suite.
+---
+
+# Index Moonbeam with Moralis
+
+## Introduction {: #introduction }
+
+As a one-stop solution for blockchain development, [Moralis](https://moralis.io/) offers a comprehensive platform that empowers developers to create, launch, and scale decentralized applications (dApps) with ease. This guide will show you how to access the API endpoints for Moonbeam using curl commands and JavaScript and Python snippets.
+
+Moralis' offerings are structured into four primary categories of API services:
+
+- **EVM API**: EVM API allows developers to query essential data for Ethereum Virtual Machine (EVM)-compatible blockchains. Moralis provides dedicated APIs for four key areas: NFTs, tokens, wallets, and general blockchain information.
+
+- **Streams API**: The Streams API enables developers to listen for on-chain events, such as smart contract event emissions, in real-time. Popular use cases include real-time wallet notifications, asset monitoring, and gaming event notifications.
+
+- **RPC API**: The RPC API offers a secure and reliable connection to various blockchain networks. It provides a high-performance gateway for developers to interact with blockchain nodes, ensuring stable and efficient communication between dApps and the blockchain.
+
+- **Auth API**: Moralis's Auth API offers a secure method for implementing wallet-based authentication in dApps. This service simplifies the often complex process of verifying user identities and managing sessions in a decentralized context.
+
+--8<-- 'text/_disclaimers/third-party-content-intro.md'
+
+## Checking prerequisites {: #checking-prerequisites }
+
+To access the endpoint, a Moralis account and API key are required.
+
+Head over to the [signup page](https://admin.moralis.io/register) to create an account. Once you're in, navigate to the API keys section. There, you can generate your own unique API key and be sure to copy it for future use.
+
+![Moralis API key](/images/builders/integrations/indexers/moralis/moralis-1.webp)
+
+## Querying Moralis API {: #querying-moralis-api }
+
+With the API key, you can try Moralis' REST APIs. The following examples show how the EVM API works:
+
+Get the token balance of a wallet:  
+
+```bash
+curl --request GET \
```

Comment:
can you please format these? the `--url` and `--header` lines should be vertically aligned